part2/stages/Commit-main

```diff
@@ -34,6 +34,8 @@ MLE_FUNC=/usr/lib/openxt/ml-functions
 [ ! -r ${TPM_FUNC} ] || . ${TPM_FUNC}
 [ ! -r ${MLE_FUNC} ] || . ${MLE_FUNC}
 [ ! -r ${MEASURED_LAUNCH_CONF} ] || . ${MEASURED_LAUNCH_CONF}
+[ ! -r ${DISK_CONF} ] || . ${DISK_CONF}
+[ ! -r ${KEY_FUNC} ] || . ${KEY_FUNC}
```

Comment:
You're matching the existing code, so this change it fine.  But I'd prefer a more positive:
[ -r ${KEY_CONF} ] && . ${KEY_CONF} style

Reply:
SGTM, I guess the other things above should might as well be converted to that style too. Thoughts?

Reply:
Your change is good since it matches the existing still. If you want to add a patch the switch them, that is up to you.